src/handlers/ens.ts

```diff
@@ -269,16 +269,23 @@ export const fetchSuggestions = async (
           uniqueId: ensDomain?.resolver?.addr?.id || ensDomain.name,
         }))
         .filter((domain: any) => !domain?.nickname?.includes?.('['));
-      suggestions = sortBy(ensSuggestions, domain => domain.nickname.length, [
-        'asc',
-      ]);
+      suggestions = ensSuggestions.sort(sorterByNicknameLengthASC);
     }
     setSuggestions(suggestions);
     setIsFetching(false);
 
     return suggestions;
   }
 };
+const sorterByNicknameLengthASC = (a: any, b: any) => {
+  if (a.nickname.length < b.nickname.length) {
```

Comment:
Are we 100% sure `nickname` is never null or undefined? If so, that's fine. If's not we might wanna use optionals here

Reply:
I didn't see such values when I replaced them and tested them on different accounts. But just in case, I will use optionals

Reply:
When I replaced `sort` and tested it on different accounts, I didn't see such values. But just in case, I will use optionals